<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotosGroupingDialog</class>
 <widget class="QDialog" name="PhotosGroupingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photos grouping tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableView" name="photosList"/>
     <widget class="QWidget" name="bottomContainer">
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="255,0">
       <item>
        <widget class="QWidget" name="optionsPart" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="horizontalLayout" stretch="0,0,0,0,255">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="groupingTypeLabel">
              <property name="text">
               <string>Grouping type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="groupingType">
              <item>
               <property name="text">
                <string>Animation</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HDR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Collage</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="optionsWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="animationOptions">
             <layout class="QHBoxLayout" name="formLayout2" stretch="0">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QDoubleSpinBox" name="speedSpinBox">
                    <property name="minimum">
                     <double>0.010000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1000.000000000000000</double>
                    </property>
                    <property name="value">
                     <double>10.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="fpsLabel">
                    <property name="text">
                     <string>fps</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="delayLabel">
                  <property name="text">
                   <string>Delay before restart:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="scaleLabel">
                  <property name="text">
                   <string>Scale:</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QCheckBox" name="stabilizationCheckBox">
                  <property name="text">
                   <string>Stabilize image</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QSpinBox" name="scaleSpinBox">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>1000</number>
                    </property>
                    <property name="value">
                     <number>10</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="percentLabel">
                    <property name="text">
                     <string notr="true">%</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QSpinBox" name="delaySpinBox">
                    <property name="maximum">
                     <number>60000</number>
                    </property>
                    <property name="singleStep">
                     <number>100</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="msLabel">
                    <property name="text">
                     <string>ms</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="speedLabel">
                  <property name="text">
                   <string>Animation speed:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Format:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="formatComboBox">
                  <item>
                   <property name="text">
                    <string notr="true">GIF</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">MNG</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="hdrOptions"/>
            <widget class="QWidget" name="collageOptions">
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>height:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="collageHeight">
                <property name="text">
                 <string notr="true">1024</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="operationName">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="generationProgressBar">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="previewButtons">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="generate">
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="previewButton">
                <property name="text">
                 <string>Generate</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="cancel">
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>172</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="cancelButton">
                <property name="text">
                 <string>Cancel</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="previewPart" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="peviewLabel">
            <property name="text">
             <string>Preview:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QScrollArea" name="resultPreview">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>64</height>
               </size>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="previewScaleSlider">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PhotosGroupingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PhotosGroupingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupingType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>optionsWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
